sdk/fleet/Azure.ResourceManager.ContainerServiceFleet/src/autorest.md

```diff
@@ -77,4 +77,27 @@ acronym-mapping:
   URI: Uri
   Etag: ETag|etag
 
+operations-to-lro-api-version-override:
```

Comment:
This will fix the version for these lro operations, is this what you expected?

Reply:
yes, this is required because for LRO we use the AKS API verison, which is different than our Fleet Swagger version.